source/includes/examples/_cart-pole.html.md

```diff
@@ -35,7 +35,8 @@ The schema `Action` names a record — `action` —  and assigns it a constraine
 
 ```inkling
 schema CartPoleConfig
-  Int8 episode_length,
+  # Note: this is a signed Int8 because -1 is used for "run until pole drops"
```

Comment:
Previously I have taken code comments like this and put them in the left pane to describe what is happening.

So instead it could say in the left pane "The schema CartPoleConfig names two records — episode_length and deque_size — and assigns each of them a type. Note, this is a signed Int8 because -1 is used for "run until pole drops" when called in the curriculum."